foundry.toml

```diff
@@ -14,8 +14,9 @@ remappings = [
 additional_compiler_profiles = [{ name = "via_ir", optimizer_runs = 10000, via_ir = true }]
 
 compilation_restrictions = [
-  { paths = "src/MinimalDelegationEntry.sol", via_ir = true },
-  { paths = "src/MinimalDelegation.sol", via_ir = true }
+  { paths = "src/CaliburEntry.sol", via_ir = true },
+  { paths = "src/Calibur.sol", via_ir = true },
+  { paths = "test/ERC7739.t.sol", via_ir = true }
```

Comment:
Why did this get added with via-ir?

Reply:
It was giving me stack too deep at some point